vercel/public/sakura.css

```diff
@@ -0,0 +1,260 @@
+/* Sakura.css v1.4.1
+ * ================
+ * Minimal css theme.
+ * Project: https://github.com/oxalorg/sakura/
+ */
```

Comment:
Since this has been modified by us, we might want to add a comment here just saying so. We don't *need* to, but it might be helpful for someone stumbling on this.